python/pyarrow/compute.py

```diff
@@ -333,6 +333,68 @@ def _make_global_functions():
 _make_global_functions()
 
 
+def dictionary_decode(arr, memory_pool=None):
```

Comment:
Many of the compute functions are automatically described from the function registry (through some kind of python metaclass magic).  I don't really know how this works.  Maybe we still need to define this method since it is a meta function?  But I think it might work even if you don't define this method.  Can you try removing this method and see if the code still works?

CC @jorisvandenbossche or @bkietz who might know more details.

Reply:
Do you mean although we don't define this function, but we can call dictionary_decode in Python directly? I will try

Reply:
It indeed work without the definition